doesn't support node 0.10

looks like `mz`/`thenify` broke our node 0.10 support:

```
stephenmathieson at MBP in ~/repos/github.com/somebody/something
$ tree . -L 2
.
├── Makefile
├── index.js
├── node_modules
│   ├── duo
│   └── duo-test
├── package.json
└── test
    └── index.js

4 directories, 4 files
stephenmathieson at MBP in ~/repos/github.com/somebody/something
$ node -v
v0.10.31
stephenmathieson at MBP in ~/repos/github.com/somebody/something
$ make
node_modules/.bin/duo --type js --root . < test/index.js > build.js

     building : from stdin
        using : component_compat
        using : stoj
    installed : component-assert@0.5.0
    installed : yields-fmt@0.0.2
    installed : component-stack@0.0.1
    installed : jkroso-equals@1.0.0
    installed : jkroso-type@1.1.0
        built : from stdin

node_modules/.bin/duo-test phantomjs --reporter spec
The file "/Users/stephenmathieson/repos/github.com/somebody/something/node_modules/duo-test/node_modules/koa-static/node_modules/koa-send/node_modules/mz/node_modules/thenify/index.js" requires `Promise`,
but neither `bluebird` nor the native `Promise` implementation were found.
Please install `bluebird` yourself.
make: *** [test] Error 1
stephenmathieson at MBP in ~/repos/github.com/somebody/something
$ nvm use 0.11
Now using node v0.11.14
stephenmathieson at MBP in ~/repos/github.com/somebody/something
$ make clean test
rm -rf build.js components
node_modules/.bin/duo --type js --root . < test/index.js > build.js

     building : from stdin
        using : component_compat
        using : stoj
    installed : component-assert@0.5.0
    installed : yields-fmt@0.0.2
    installed : component-stack@0.0.1
    installed : jkroso-equals@1.0.0
    installed : jkroso-type@1.1.0
        built : from stdin

node_modules/.bin/duo-test phantomjs --reporter spec

thing
  ✓ should do stuff 

  1 passing (11ms)

stephenmathieson at MBP in ~/repos/github.com/somebody/something
```


where my **Makefile** looks like:

```makefile
BIN := node_modules/.bin
DUO := $(BIN)/duo
TEST := $(BIN)/duo-test

test: build.js node_modules
	$(TEST) phantomjs --reporter spec

build.js: test/index.js index.js node_modules
	$(DUO) --type js --root . < $< > $@

node_modules: package.json
	npm install
	@touch $@

clean:
	rm -rf build.js components
```

--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "name": "duo-test",
   "version": "0.3.13",
   "dependencies": {
+    "bluebird": "^2.9.6",
     "buffer-events": "0.0.2",
     "co": "^3.1.0",
     "co-exec": "^1.1.0",

